pinot-controller/src/main/resources/app/utils/PinotMethodUtils.ts

```diff
@@ -272,15 +272,13 @@ const getQueryResults = (params) => {
     // if sql api throws error, handle here
     if(typeof queryResponse === 'string'){
       errorStr = queryResponse;
-    } else if (queryResponse && queryResponse.exceptions && queryResponse.exceptions.length) {
+    } 
+    if (queryResponse && queryResponse.exceptions && queryResponse.exceptions.length) {
       errorStr = JSON.stringify(queryResponse.exceptions, null, 2);
```

Comment:
Let's start displaying the message instead of the message body JSON? @npawar will that work? I assume this is not intentional.

If intentional, let's add try/catch here. Any error will cause a white screen.

Reply:
wouldnt say it's intentional.But i'm guessing it is done so, because 1) there can be multiple exceptions in this list 2) we also want to display error code. But if a better UX is possible here without losing all these, that works